endpoints/bus/realtime.js

```diff
@@ -56,36 +80,29 @@ const getBusRoutes = (data) => new Promise((resolve, reject) => {
         return reject(e)
       }
 
-      var routes = data.routes || []
-
-      var objRoutes = {
-        results: []
-      }
-      routes.forEach(function (route, key) {
+      var busses = data.positions || []
 
+      requestedRoutes.forEach(function (route) {
         var objRoute = {
-          busNr: route.id || '', // will be undefined if none are active
+          busNr: route,
           busses: []
         }
-        objRoutes.results.push(objRoute)
-
-        if (!route.busses) return // No busses active, eg. after schedule
-
-        route.busses.forEach(function (bus, key) {
-
-          var location = isn2wgs(bus.X, bus.Y),
-            oneRoute = {
-            'unixTime': Date.parse(bus.TIMESTAMPREAL) / 1000,
-            'x': location.latitude,
-            'y': location.longitude,
-            'from': bus.FROMSTOP,
-            'to': bus.TOSTOP
+        busses.forEach(function (bus) {
+          if (bus.routeNumber === route) {
+            objRoute.busses.push({
+              'unixTime': Math.floor(bus.gpsTime / 1000),
+              'x': bus.lat,
+              'y': bus.lon,
+              'from': bus.lastStop,
+              'to': bus.nextStop
```

Comment:
`from` and `to` used to be the names of the stops, but they are now some ID numbers. e.g. `90000635` and `90000366`. That's a breaking change, but better than nothing...